doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -284,155 +294,81 @@ Induction
       induction n.
       exact (le_n 0).
 
-.. exn:: Not an inductive product.
-   :undocumented:
-
-.. exn:: Unable to find an instance for the variables @ident ... @ident.
+.. example:: :tacn:`induction` with :n:`@occurrences`
```

Comment:
That's not a very convincing example since it doesn't behave differently than without the `@occurrences` clause.

Here is an example where it does behave differently:

```coq
Lemma induction_test : forall n:nat, n = n -> n <= n.
intros.
induction n in H |-.
Show 2.
```

However, I have no idea if there exists a *useful* example of this clause with `induction`.

Reply:
If it's not useful then maybe drop the example and also consider whether to drop `occurrences` from the syntax.

Reply:
> also consider whether to drop `occurrences` from the syntax

I guess that it should be kept for uniformity with `destruct` (where it can be more clearly useful). And there might be use cases that I'm neglecting.